src/components/Navbar.tsx

```diff
@@ -8,17 +13,24 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <Link
+            className={classNames('navbar-item ', {
+              'has-background-grey-lighter': pathname === '/',
+            })}
+            to="/"
+          >
             Home
-          </a>
+          </Link>
 
-          <a
+          <Link
             aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
+            className={classNames('navbar-item ', {
+              'has-background-grey-lighter': pathname === '/people',
+            })}
+            to="people"
```

Comment:
The `to` attribute for the 'People' link should be `/people` instead of `people` to ensure the correct path is used for navigation. This ensures the link navigates to the correct route.